inst/shiny/modules/tab_tlg.R

```diff
@@ -1,96 +1,83 @@
+.TLG_DEFINITIONS <- {
```

Comment:
this is probably the most obscure object (in terms of its names) that i've ever seen to this day :)